lib/compat/wordpress-6.9/template-activate.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+// How does this work?
+// 1. For wp_template, we remove the custom templates controller, so it becomes
+//    a normal posts endpoint, modified slightly to allow auto-drafts.
```

Comment:
I think this is the right thing to do but I'm a bit concerned about potential breaking changes.

Reply:
Let's add a fallback for a slug id in a follow-up as discussed. Right there should be a summary. The main change is that whatever imitation of the post type endpoint there was for `wp_template` is now replaced with a real post type endpoint. And another one is added to handle theme/plugin templates separately.